src/OpenTelemetry.AutoInstrumentation.StartupBootstrapper/OpenTelemetry.AutoInstrumentation.StartupBootstrapper.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Do the dependencies in this project (especially the aspnetcore dependencies) automatically resolve correctly for newer target frameworks (net6 and net7)?

Reply:
Yes, new framework will automatically resolve it. We don't need to maintain the hosting library for every framework.